transformer/unflatten.go

```diff
@@ -0,0 +1,64 @@
+package transformer
+
+import (
+	"sort"
+	"strings"
+
+	"github.com/telenornms/skogul"
+)
+
+type Unflatten struct {}
+
+// Transform created a container of metrics
+func (u *Unflatten) Transform(c *skogul.Container) error {
+	for mi := range c.Metrics {
+		c.Metrics[mi] = u.convertValues(c.Metrics[mi])
+	}
+
+	return nil
+}
+
```

Comment:
Given the complexity of coverteValues, a human-readable description of the algorithm would be helpful here for future debugging.